### DISCUSSION TITLE
Kopfzeile nicht sichtbar trotz \pagestyle{tudheadings}

### DISCUSSION THREAD

**Discussion by Arduryo** [tud-cd/tudscr#97] [Q&A] [answered]
Hallo, ich habe derzeit das Problem, dass trotz des Workarounds in #59 und der Lösung in #63 die Kopfzeile nicht angezeigt wird. Ich vermute, dass die KOMA-Skripte die Einstellungen überschreiben.

Das MWE sieht bei mir so aus: 
```
\documentclass[ngerman,BCOR=4mm, cd=lightcolor, twoside]{tudscrreprt}
\usepackage[automark]{scrlayer-scrpage}
\pagestyle{headings}
\begin{document}
\chapter{...}
\end{document}
```

Mit der Nutzung von \pagestyle{tudheadings} bzw. \pagestyle{plain.tudheadings} produziert er auf den chapter-Seiten immer die auf der Titelseite verwendete Kopfzeile mit Logo und prägnantem Balken. Ab Seite zwei möchte ich aber nur den chapter-Titel und die Seitenzahl stehen haben. 
Die Verwendung von cdhead-Einstellungen, wie im Benutzerhandbuch beschrieben, brachten bisher auch keinen Erfolg.

Viele Grüße

**Comment by mrpiggi** [comment]
Funktioniert bei mir problemlos:
```latex
\documentclass[ngerman,BCOR=4mm, cd=lightcolor, twoside]{tudscrreprt}
\usepackage{blindtext}
\begin{document}
\Blinddocument
\end{document}
```
Was genau erwartest du denn für ein Ergebnis?
![test-01](https://github.com/tud-cd/tudscr/assets/7372826/36e4d9df-28ce-4da0-9de0-65f67f91ea97)
![test-02](https://github.com/tud-cd/tudscr/assets/7372826/272a5825-cbb3-4215-b5d4-b2f0fa98fa65)
![test-03](https://github.com/tud-cd/tudscr/assets/7372826/971197d7-aa34-4b60-98a7-cd01a628dab6)

**Comment by keinstein** [comment, marked as answer]
Typischerweise werden Kapitel auf Seiten ohne Kolumnentitel gesetzt. Offensichtlich auch hier. Definiert man `\chapterpagestyle` um, sollten die Kapitel auch mit Kolumnentiteln gesetzt werden. Beachten sollte man dabei, dass per default rechts die Abschnittsüberschrift gesetzt wird, die durch \chapter genullt wird. Beispiel:
```
\documentclass[ngerman,BCOR=4mm, cd=lightcolor, twoside,headsepline=true]{tudscrreprt}
\usepackage[automark]{scrlayer-scrpage}
\pagestyle{headings}
\usepackage{blindtext}
\begin{document}
\def\chapterpagestyle{headings}
\chapter{...}
\strut
\chapter{...}
\strut
\Blinddocument
\thispagestyle{plain}
\Blinddocument
\end{document}
```

Fazit: Um die korrekte Funktion zu überprüfen muss man sich schon ganz schön anstrengen. Hier wird die Gegenprobe auf Seite 16 gemacht. Die Linie verschwindet, damit kann Seite 1 nicht mit `plain` gesetzt sein.

Edit: klappt wirklich mit plain.

**Comment by mrpiggi** [comment]
Sehr nett von dir @keinstein, dass du dir die Mühe gemacht hast, etwas mehr Hilfestellung zu geben. Ich hätte das auch getan, wenn von @Arduryo eine Rückmeldung gekommen wäre, was genau denn erreicht werden soll. Da aber quasi nichts spezifiziert wurde und die Kopfzeilen per se funktionieren, warte ich gerne noch auf eine Antwort...

**Comment by Arduryo** [comment]
Hallo, danke für die Antworten und sorry für die späte Rückmeldung, ich hatte zwischendurch mein TeX komplett gecrasht und musste erstmal diese Probleme lösen.
Tatsächlich funktionieren sie auf den Seiten ohne Kolumnentitel. Da der Lehrstuhl keine Aussage dazu gemacht hat, wie er es haben möchte, wollte ich ausprobieren, wie es auf Seiten mit Kolumnentitel aussieht. Den Workaround von @keinstein kannte ich noch nicht, die Um-Definierung von \chapterpagstyle war mir nicht in den Sinn gekommen.

Jetzt funktioniert alles, danke für die Hilfe :)
